include/core/parameters.h

```diff
@@ -431,6 +461,16 @@ namespace Parameters
       phase_change
     } thermal_expansion_model;
 
+    enum class TracerDiffusivityModel
+    {
+      constant,
+      tanh_levelset
+    } tracer_diffusivity_model;
```

Comment:
That's not really a diffusivity model per say right, it's more like an interface model and we have classes for interface models (e.g. surface tension is an interface model)